rancid-tomatillos/src/App.js

```diff
@@ -9,24 +9,45 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
+      movies: []
     }
   }
 
-  displayMovieInfo = (id) => {
-    const filteredMovie = this.state.movies.filter(movie => movie.id === id);
+  componentDidMount = () => {
+    fetch("https://rancid-tomatillos.herokuapp.com/api/v2/movies")
+      .then(response => {
+        if (!response.ok) {
+          throw new Error(`Error: ${response.status}`)
+        } else {
+          return response.json();
+        }
+      })
+      .then(data => {
+        this.setState({ movies: data.movies })
+      })
+      .catch(err => console.log(err))
```

Comment:
Nice work! To make this even better, ensure that you take advantage of the error that's coming back from the API. Instead of just console logging it, consider providing some feedback to your users.